doc/source/io.rst

```diff
@@ -279,21 +324,29 @@ Write to a raster file using `concurrent.futures` and the `geowombat` :func:`to_
             n_chunks=200    # number of window chunks to send as concurrent futures
         )
 
-Write to a raster file using `Dask` and the `geowombat` :func:`save` function.
+Write to a VRT file
```

Comment:
This is new.

Reply:
I love that you are making the documentation more beginner friendly!